src/test/resources/unsolvedparameter/input/com/example/Simple.java

```diff
@@ -0,0 +1,13 @@
+package com.example;
+
+import com.github.javaparser.ast.NodeList;
+import com.github.javaparser.ast.ImportDeclaration;
+
+public class Simple {
+
+    public void lengthImportStatement(NodeList<ImportDeclaration> listOfImports) {
+        for (ImportDeclaration importDecl: listOfImports) {
```

Comment:
Please create a version of this test case without the for loop, to make sure that `ImportDeclaration` is still included even if its only use is in a type variable instantiation.